### PR TITLE
Update Dockerfile with new versions

### PR DESCRIPTION
**Pull request by c-vandendyck-kbr**
**Update ENV:**

- solrwayback 5.1
- tomcat 9
- solr 9

Also update RUN commands in order to call the good folder and the `-c` option

### CODE REVIEW

**Comment by thomasegense** [comment]
Thanks. I will look into it tomorrow!

**Comment by c-vandendyck-kbr** [comment]
Hi @thomasegense !

And sorry it took me so much time ;-)

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
This is indeed working as it should. Thanks a lot! The Docker file contains documentation on how to start the service in the lines 1-22. I would like the following changes here:

- Could you please update exports in line 10-12 to match the ENV set in the file.
- In line 13 add -c to the command.
- in line 14 change apache-tomcat to tomcat, as thats the name of the directory now.

**Comment by VictorHarbo** [comment]
Ps. I've tested the Dockerfile on a Macbook running an M1 processor and it works flawlessly.

**Comment by thomasegense** [comment]
@VictorHarbo  When you accept the PR, also close this Jira: https://github.com/netarchivesuite/solrwayback/issues/439

**Comment by c-vandendyck-kbr** [comment]
Hi @VictorHarbo !

Thanks for your feedback!
In fact, in the excitement of this first pull-request for the solrwayback project, I completely forgot about the documentation. 
My mistake !

I've just corrected it ;-)

File: Dockerfile [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
 
 # When the container is running, run the following commands to start Solr and Tomcat:
-# export SOLRWAYBACK_VERSION=4.4.2
-# export APACHE_TOMCAT_VERSION=8.5.60
-# export SOLR_VERSION=7.7.3
-# ./unpacked-bundle/solrwayback_package_$SOLRWAYBACK_VERSION/solr-$SOLR_VERSION/bin/solr start
-# ./unpacked-bundle/solrwayback_package_$SOLRWAYBACK_VERSION/apache-tomcat-$APACHE_TOMCAT_VERSION/bin/startup.sh
+# export SOLRWAYBACK_VERSION 5.1.0
+# export APACHE_TOMCAT_VERSION 9
+# export SOLR_VERSION 9
```
Comment (VictorHarbo) [inline Dockerfile:12]: Sorry. these are bash variables, they need a =-sign in between variable and value withoug spaces to work.

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Single change and then it is good  to go :)

**Comment by VictorHarbo** [comment]
> Hi @VictorHarbo !
> 
> Thanks for your feedback! In fact, in the excitement of this first pull-request for the solrwayback project, I completely forgot about the documentation. My mistake !
> 
> I've just corrected it ;-)

No worries. We love to have excited people contributing! One minor change and I'll merge it.

File: Dockerfile (hunk 1 above)
Comment (c-vandendyck-kbr) [reply Dockerfile]: Indeed! Done

**Review by VictorHarbo — APPROVED** [review]
Thank you @c-vandendyck-kbr. Everything looks good now. Congratulations on your first contribution to SolrWayback!

**Pull request merged**